src/index.html

```diff
@@ -20,8 +20,372 @@
       href="styles/index.scss"
     />
   </head>
-
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <a
+        href="#Home"
+        class="header__image"
+      >
+        <img
+          class="header__logo"
+          src="images/logo.png"
+          alt="logo Moyo"
+        />
+      </a>
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__li">
```

Comment:
Try to be more semantic when describing you BEM classes. You can use `nav__item` instead of `nav__li` for example.

Reply:
Modification completed